packages/backend/types/storage/index.ts

```diff
@@ -0,0 +1 @@
+export { StorageHandler } from '@aws-amplify/backend-function';
```

Comment:
So this is not a trigger or resolver? So we want to have it as `@aws-amplify/backend/types/storage`

Reply:
That's a question I need to ask @josefaidt @sobolk @edwardfoyle

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html

Reply:
Done! added as a trigger